spec/helpers/library_helper_spec.rb

```diff
@@ -10,6 +10,6 @@
 #     end
 #   end
 # end
-describe LibraryHelper do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
+# describe LibraryHelper do
```

Comment:
These are sample codes given by generator as a tip to what you should put here. Please add some useful code here. This need not be a commit as such! :)